documentation.html

```diff
@@ -0,0 +1,114 @@
+---
+layout: page
+title: OpenMM Documentation
+---
+<div class="jumbotron jumbtron-color">
+  <div class="container">
+  <div class="row clearfix">
+  <div class="col-md-12 column">
+   <!--  <a href="https://github.com/simtk/openmm"><img style="float: right; border: 0; z-index: 1000" src="https://camo.githubusercontent.com/365986a132ccd6a44c23a9169022c0b5c890c387/68747470733a2f2f73332e616d617a6f6e6177732e636f6d2f6769746875622f726962626f6e732f666f726b6d655f72696768745f7265645f6161303030302e706e67" alt="Fork me on GitHub" data-canonical-src="https://s3.amazonaws.com/github/ribbons/forkme_right_red_aa0000.png"></a>
+     -->
+      <h1>Documentation</h1>
+      <p>Resources for finding your way around OpenMM.</p>
+  </div>
+  </div>
+  </div>
+</div>
+
+<div class="container">
+  <div class="row">
+    <div class="col-md-10" role="main">
+      <div>
+        <h2 class="page-header">Getting Started</h2>
+        <p class="lead"> The easiest way to get started is to familiarize yourself with the User Guide, which contains basics about installing OpenMM, making sure that it runs smoothly, and running a couple of example simulations. </p>
+        <div class="row clearfix">
+          <div class="col-md-3 column">
+            <h3>
+              <a href="http://docs.openmm.org/7.1.0/userguide/index.html">User Guide</a>
+            </h3>
+            <p class="enlarge">
+              Start here if it's your first time using OpenMM. These documents will get you up and running simulations in short order.
+            </p>
+          </div>
+          <div class="col-md-3 column">
+            <h3>
+              <a href="http://docs.openmm.org/7.1.0/developerguide/index.html">Dev Guide</a>
+            </h3>
+            <p class="enlarge">
+              This section provides in-depth information on the code-base and how to contribute to the development of OpenMM itself.
+            </p>
+          </div>
+          <div class="col-md-3 column">
+            <h3>
+              <a href="http://docs.openmm.org/7.1.0/api-python/index.html">Python API
+              </a>
+            </h3>
+            <p class="enlarge">
+              The Python API provides information about the classes and methods available to Python developers.
+            </p>
+          </div>
+          <div class="col-md-3 column">
+            <h3>
+              <a href="http://docs.openmm.org/7.1.0/api-c++/index.html">C++ API
+              </a>
+            </h3>
+            <p class="enlarge">
```

Comment:
is this why font size seems different? all the html pages add `class="enlarge"` whereas the markdown just uses straight `p`s